README.md

```diff
@@ -207,31 +162,126 @@ Users want to determine several types of the formatting width as following
+  * `years` (`string`): The style to be used for formatting years. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `yearsDisplay` (`string`): When to display years. `"auto"` means that years won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `years` is `undefined` and `"always"` otherwise.
+  * `months` (`string`): The style to be used for formatting months. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `monthsDisplay` (`string`): When to display months. `"auto"` means that months won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `months` is `undefined` and `"always"` otherwise.
+  * `weeks` (`string`): The style to be used for formatting weeks. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `weeksDisplay` (`string`): When to display weeks. `"auto"` means that weeks won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `weeks` is `undefined` and `"always"` otherwise.
+  * `days` (`string`): The style to be used for formatting days. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `daysDisplay` (`string`): When to display days. `"auto"` means that days won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `days` is `undefined` and `"always"` otherwise.
+  * `hours` (`string`): The style to be used for formatting hours. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `hoursDisplay` (`string`): When to display hours. `"auto"` means that hours won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `hours` is `undefined` and `"always"` otherwise.
+  * `minutes` (`string`): The style to be used for formatting minutes. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `minutesDisplay` (`string`): When to display minutes. `"auto"` means that minutes won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `minutes` is `undefined` and `"always"` otherwise.
+  * `seconds` (`string`): The style to be used for formatting seconds. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `secondsDisplay` (`string`): When to display seconds. `"auto"` means that seconds won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `seconds` is `undefined` and `"always"` otherwise.
+  * `milliseconds` (`string`): The style to be used for formatting milliseconds. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
```

Comment:
Should we adopt `"numeric"` as suggested in https://github.com/tc39/proposal-intl-duration-format/issues/64#issuecomment-892814950 ?

Reply:
@sffc I'd missed them out, check the README now.

Reply:
We may need to update this again once #64 is fully resolved, but that's only tinkering around the edges.  This looks good to me for merging.